samples/java-reliable-timers/src/it/java/com/example/OrderIntegrationTest.java

```diff
@@ -0,0 +1,63 @@
+package com.example;
+
+import com.example.domain.OrderDomain;
+import com.google.protobuf.Empty;
+import kalix.javasdk.testkit.junit.KalixTestKitResource;
+import org.junit.ClassRule;
+import org.junit.Test;
+
+import static java.util.concurrent.TimeUnit.*;
+
+// This class was initially generated based on the .proto definition by Kalix tooling.
+//
+// As long as this file exists it will not be overwritten: you can maintain it yourself,
+// or delete it so it is regenerated as needed.
+
+// Example of an integration test calling our service via the Kalix proxy
+// Run all test classes ending with "IntegrationTest" using `mvn verify -Pit`
+public class OrderIntegrationTest {
+
+  /**
+   * The test kit starts both the service container and the Kalix proxy.
+   */
+  @ClassRule
+  public static final KalixTestKitResource testKit =
+    new KalixTestKitResource(Main.createKalix());
+
+  /**
+   * Use the generated gRPC client to call the service through the Kalix proxy.
+   */
+  private final OrderService client;
+
+  public OrderIntegrationTest() {
+    client = testKit.getGrpcClient(OrderService.class);
+  }
+
+  @Test
```

Comment:
```suggestion
  @Test
  @Ignore("to be implemented")
```

Reply:
We add it to the codegen, but only for unit tests. I will create an issue to track it.